revenue/revenue.go

```diff
@@ -56,91 +47,68 @@ func GetRevenueReport(cfg *Config) (*Report, error) {
 		channelIDs[id] = closedChannel.ChannelPoint
 	}
 
-	events, err := getEvents(channelIDs, cfg.ForwardingHistory)
+	fwds, err := cfg.ForwardingHistory()
 	if err != nil {
 		return nil, err
 	}
 
+	events := getEvents(channelIDs, fwds)
 	return getReport(events), nil
 }
 
-// getEvents gets calls the paginated query function until it has all the
-// forwarding events for the period provided. It takes a map of shortChannelIDs
-// to outpoints which is used to convert forwarding events short ids to
-// outpoint strings.
-func getEvents(channelIDs map[lnwire.ShortChannelID]string, queryForwards func(
-	offset, maxEvents uint32) (*lndclient.ForwardingHistoryResponse,
-	error)) ([]revenueEvent, error) {
+// getEvents produces a set of revenue events from a set of forwarding events
+// that can be used to create a report.
+func getEvents(channelIDs map[lnwire.ShortChannelID]string,
```

Comment:
Nice that we can use the code in `lndwrap` to do the actual querying. But now that the meaning of this function has changed, maybe rename it to `toRevenueEvents()` or something to that effect?